test/Succeed/LevelUnivFunSorts.agda

```diff
@@ -0,0 +1,17 @@
+{-# OPTIONS --level-universe #-}
+
+open import Common.Level
+open import Agda.Primitive
+
+_ : LevelUniv
+_ = Level
+_ : LevelUniv
+_ = Level -> Level
+_ : Setω2
+_ = Level -> Setω1
+_ : Setω
+_ = Level -> Set
+_ : Setω2
+_ = Setω1 -> Level
+-- should this really work ?
```

Comment:
I think this shouldn't work, since the main goal of `LevelUniv` would be to prevent functions into `Level` (see my comment on the `funSort'` function).